### PR TITLE
Issue #36386

### PR DESCRIPTION
**Pull request by YashPimple**

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/user-namespaces.md [hunk 1]
```diff
@@ -91,8 +91,9 @@ host. Inside the container the process will think it is running as root (and
 therefore tools like `apt`, `yum`, etc. work fine), while in reality the process
 doesn't have privileges on the host. You can verify this, for example, if you
 check the user the container process is running `ps` from the host. The user
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/user-namespaces.md:93]: > You can verify this, for example, if you check the user the container process is running `ps` from the host.

This sentence doesn't make sense.

File: content/en/docs/concepts/workloads/pods/user-namespaces.md [hunk 2]
```diff
@@ -91,8 +91,9 @@ host. Inside the container the process will think it is running as root (and
...
-`ps` shows is not the same as the user you see if you execute inside the
-container the command `id`.
+`ps` shows is not the same as the user you see if the command `id` is execute
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/user-namespaces.md:94]: "The user `ps` shows is not ..."

This text is confusing.

**Comment by divya-mohan0209** [comment]
Hello @YashPimple, please amend the commit & retitle PR with the right message. We do not accept issue numbers in the title/commit messages & this typically needs to be included in the description of the PR.

Additionally, please address comments by @tengqm

File: content/en/docs/concepts/workloads/pods/user-namespaces.md (hunk 1 above)
Comment (YashPimple) [reply content/en/docs/concepts/workloads/pods/user-namespaces.md:93]: Sure I will change it.

**Comment by sftim** [comment]
/hold

with https://github.com/kubernetes/website/pull/36458#issuecomment-1233822803 in mind. OK to unhold once the PR title _and_ commit message are updated.

@YashPimple it's OK to ask for help with making those changes.

**Comment by YashPimple** [comment]
@sftim In which format   do i need to name the branch?

**Comment by divya-mohan0209** [comment]
@YashPimple : Please amend the title + the commit message to what exactly you're fixing without mentioning the issue number. The issue number can be mentioned within the description.

**Comment by sftim** [comment]
/hold
The PR title isn't right. @YashPimple, please wait for a reviewer to check it's OK to unhold.

**Comment by Sea-n** [comment]
The PR author have opened another PR instead.

- #36592

/close

**Pull request closed without merging**